test/unit/controllerSpec.js

```diff
@@ -48,5 +50,15 @@ describe('Concept dictionary controllers', function() {
     		  [{name: 'Anatomy', description: 'Anatomic sites / descriptors', uuid: 'ad491c7a-c2cc-11de-8d13-0010c6dffd0f'},
                {name: 'Procedure', description: 'Describes a clinical procedure', uuid: 'bd490bf4-c2cc-11de-8d13-0010c6dffd0f'}]);
     });
+    
+    it('should send delete request at uuid adress extracted from selection map', function() {
+    	//scope.adress equals Util.getOpenmrsUrl()
+        $httpBackend.expectDELETE(scope.adress+'/ws/rest/v1/conceptclass/ad491c7a-c2cc-11de-8d13-0010c6dffd0f?').respond("DELETE SUCCESS")
```

Comment:
Have you seen https://docs.angularjs.org/api/ngMock/function/angular.mock.inject ? Just add inject(function(Util) { ... })